apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.csproj

```diff
@@ -2,8 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Version>3.0.0-beta13</Version>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netstandard2.0</TargetFrameworks>
```

Comment:
It feels odd for a library to target .NET Core App rather than .NET Standard. Is that definitely necessary? I'd expect netstandard2.1 for .NET Core 3.0 support.

Reply:
I know ASP.NET Core 3 will only run on .NET Core, but I'd still expect most packages to target netstandard2.1. I guess it depends on whether we're depending on anything that targets netcoreapp3.0 directly. Currently the PR depends on Microsoft.AspNetCore.App, but I don't know whether it's best to depend on that from *libraries* as opposed to from apps themselves.

Reply:
I'm not sure if referencing Microsoft.AspNetCore.App is the right thing too. But there are no other runtime packages for ASP.NET Core which you can reference. Only the Microsoft.Extensions.* packages are published as netstandard2.0 packages for .NET Core, no the Microsoft.AspNetCore.* ones.

Reply:
Interesting - that package contains the reference assemblies, by the looks of it.

Reply:
I doubt there is a NuGet package of `Microsoft.AspNetCore.App` anymore, the last one is preview3. My understanding is that they've added ASP.NET Core to the shared framework of .NET Core 3. And you can explicitly reference it by adding the `<FrameworkReference>` element.